lib/notifier/email.js

```diff
@@ -0,0 +1,102 @@
+const _ = require('lodash/fp')
+const utils = require("./utils")
+
+const email = require('../email')
+
+const {
+  PING,
+  STALE,
+  LOW_CRYPTO_BALANCE,
+  HIGH_CRYPTO_BALANCE,
+  CASH_BOX_FULL,
+  LOW_CASH_OUT
+} = require('./codes')
+
+function alertSubject(alertRec, config) {
+  let alerts = []
+
+  if (config.balance) {
+    alerts = _.concat(alerts, alertRec.general)
+  }
+
+  _.keys(alertRec.devices).forEach(function (device) {
+    if (config.balance) {
+      alerts = _.concat(alerts, alertRec.devices[device].balanceAlerts)
+    }
+
+    if (config.errors) {
+      alerts = _.concat(alerts, alertRec.devices[device].deviceAlerts)
+    }
+  })
+
```

Comment:
Shouldn't this be like the suggestion? To avoid using the default forEach function

```suggestion
  _.forEach(_.keys(alertRec.devices), (device) => {
    if (config.balance)
      alerts = _.concat(alerts, alertRec.devices[device].balanceAlerts)

    if (config.errors)
      alerts = _.concat(alerts, alertRec.devices[device].deviceAlerts)
})
```